ci-operator/step-registry/openshift-qe/winc/openshift-qe-winc-commands.sh

```diff
@@ -0,0 +1,170 @@
+          imagePullPolicy: IfNotPresent
+          command:
+            - pwsh.exe
+            - -command
+            - \$listener = New-Object System.Net.HttpListener; \$listener.Prefixes.Add('http://*:80/'); \$listener.Start();Write-Host('Listening at http://*:80/'); while (\$listener.IsListening) { \$context = \$listener.GetContext(); \$response = \$context.Response; \$content='<html><body><H1>Windows Container Web Server</H1></body></html>'; \$buffer = [System.Text.Encoding]::UTF8.GetBytes(\$content); \$response.ContentLength64 = \$buffer.Length; \$response.OutputStream.Write(\$buffer, 0, \$buffer.Length); \$response.Close(); };
+          securityContext:
+            runAsNonRoot: false
+            windowsOptions:
+              runAsUserName: "ContainerAdministrator"
+EOF
+
+  # Wait up to 5 minutes for Windows workload to be ready
+  oc wait deployment win-webserver -n winc-test --for condition=Available=True --timeout=5m
+
+  # Create Linux workload
+  oc create -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: linux-webserver
+  labels:
+    app: linux-webserver
+spec:
+  ports:
+  # the port that this service should serve on
+  - port: 8080
+    targetPort: 8080
+  selector:
+    app: linux-webserver
+  type: LoadBalancer
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: linux-webserver
+  name: linux-webserver
+spec:
+  selector:
+    matchLabels:
+      app: linux-webserver
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: linux-webserver
+      name: linux-webserver
+    spec:
+      containers:
+      - name: linux-webserver
+        image: quay.io/openshifttest/hello-openshift:multiarch-winc
+        ports:
+        - containerPort: 8080
+EOF
+  # Wait up to 5 minutes for Linux workload to be ready
+  oc wait deployment linux-webserver -n winc-test --for condition=Available=True --timeout=5m
+}
+
+IAAS_PLATFORM=$(oc get infrastructure cluster -o=jsonpath="{.status.platformStatus.type}"| tr '[:upper:]' '[:lower:]')
+
+
+winworker_machineset_name=$(oc get machineset -n openshift-machine-api -o json | jq -r '.items[] | select(.metadata.name | test("win")).metadata.name')
+winworker_machineset_replicas=$(oc get machineset -n openshift-machine-api $winworker_machineset_name -o jsonpath="{.spec.replicas}")
+
+echo "Waiting for Windows nodes to come up"
+while [[ $(oc -n openshift-machine-api get machineset/${winworker_machineset_name} -o 'jsonpath={.status.readyReplicas}') != "${winworker_machineset_replicas}" ]]; do echo -n "." && sleep 10; done
+#oc wait machineset $winworker_machineset_name -n openshift-machine-api --for jsonpath="{.status.fullyLabeledReplicas}"=$winworker_machineset_replicas --timeout=5m
+#oc wait machine -l machine.openshift.io/cluster-api-machineset=${winworker_machineset_name} -n openshift-machine-api --for jsonpath="{.status.phase}"=Running --timeout=30m
+
+oc wait nodes -l kubernetes.io/os=windows --for condition=Ready=True --timeout=15m
```

Comment:
Are you sure that 15 minutes is enough to create Windows workers? or it is the limit in Prow?